docs/en/how-to/index.md

```diff
@@ -0,0 +1,557 @@
+```
+
+///
+
+/// tab | Windows
+
+```doscon
+(.venv) C:\...>git add some\interesting_file.py
+(.venv) C:\...>git commit -m "Minor change"
+check toml...........................................(no files to check)Skipped
+check yaml...............................................................Passed
+check for case conflicts.................................................Passed
+fix end of files.........................................................Passed
+trim trailing whitespace.................................................Passed
+```
+
+///
+
+Now you are ready to start hacking on BeeWare docs!
+
+## Building the BeeWare Tutorial
+
+### Build the Tutorial locally
+
+Once your development environment is set up, run:
+
+/// tab | macOS
+
+```console
+(.venv) $ tox -e docs
+```
+
+///
+
+/// tab | Linux
+
+```console
+(.venv) $ tox -e docs
+```
+
+///
+
+/// tab | Windows
+
+```doscon
+(.venv) C:\...>tox -e docs
+```
+
+///
+
+The output of the file should be in the `docs/_build/html` folder. If
+there are any markup problems, they'll raise an error.
+
+### Live Tutorial preview
+
+To support rapid editing of documentation, BeeWare also has a "live
+preview" mode:
+
+/// tab | macOS
+
+```console
+(.venv) $ tox -e docs-live
+```
+
+///
+
+/// tab | Linux
+
+```console
+(.venv) $ tox -e docs-live
+```
+
+///
+
+/// tab | Windows
+
+```doscon
+(.venv) C:\...>tox -e docs-live
+```
+
+///
+
+This will build the documentation, start a web server to serve the build
+documentation, and watch the file system for any changes to the
+documentation source. If a change is detected, the documentation will be
+rebuilt, and any browser viewing the modified page will be automatically
+refreshed.
+
+### Tutorial linting
+
+The build process will identify reStructuredText problems, but BeeWare
+performs some additional "lint" checks. To run the lint checks:
+
+/// tab | macOS
+
+```console
+(.venv) $ tox -e docs-lint
+```
+
+///
+
+/// tab | Linux
+
+```console
+(.venv) $ tox -e docs-lint
+```
+
+///
+
+/// tab | Windows
+
+```doscon
+(.venv) C:\...>tox -e docs-lint
+```
+
+///
+
+This will validate the documentation does not contain:
+
+- dead hyperlinks
+- misspelled words
+
+If a valid spelling of a word is identified as misspelled, then add the
+word to the list in `docs/spelling_wordlist`. This will add the word to
+the spellchecker's dictionary. When adding to this list, remember:
+
+- We prefer US spelling, with some liberties for programming-specific
+  colloquialisms (e.g., "apps") and verbing of nouns (e.g., "scrollable")
+- Any reference to a product name should use the product's preferred
+  capitalization. (e.g., "macOS", "GTK", "pytest", "Pygame",
+  "PyScript").
+- If a term is being used "as code", then it should be quoted as inline
+  code rather than being added to the dictionary.
+
+### Building translations
+
+The BeeWare tutorial is available in multiple languages, which you can
+build individually or all at the same time.
```

Comment:
Do we need details about updating the translations first if you've modified English content?